plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -50,6 +50,12 @@
     private Duration serviceCacheTtl = new Duration(3, MINUTES);
     private boolean queryResultsCacheEnabled;
 
+    private int rpcInitialChannelCount = 1;
+    private int rpcMaxChannelCount = 2;
+    private int rpcMinChannelCount = 1;
+    private int minRpcPerChannel = 1;
+    private int maxRpcPerChannel = 8;
```

Comment:
How did you choose these values? The default values in the library builder are:
* rpcInitialChannelCount: 1
* rpcMaxChannelCount: 1
* rpcMinChannelCount: 1
* minRpcPerChannel: 0
* maxRpcPerChannel: 2147483647

The commit title "Add configurable dynamic channel pool in BigQuery connector" looks little misleading because it changes the default setting if my understanding is correct.